ResourceManager/Strings.cs

```diff
@@ -27,10 +27,9 @@ public sealed class Strings : Translate
 
 Yes, I allow telemetry!");
 
-        private readonly TranslationString _installGitInstructions = new TranslationString("Install git...");
-        private readonly TranslationString _findGitExecutable = new TranslationString("Find git...");
-        private readonly TranslationString _gitExecutableNotFoundText =
-            new TranslationString("The Git executable could not be located on your system.");
+        private readonly TranslationString _installGitInstructions = new TranslationString("Install Git\r\nGo to GitExtensions wiki");
+        private readonly TranslationString _findGitExecutable = new TranslationString("Browse to Git\r\n Find Git on your computer");
+        private readonly TranslationString _gitExecutableNotFoundText = new TranslationString("Git Extensions requires a Git executable to be on your computer which does not appear to be the case. Please install Git or click Browse to find Git on your computer manually.");
```

Comment:
These many words don't make it more the dialog user friendly. In fact I feel it is the opposite.

```
The Git executable could not be located on your system.
```
vs
```
Git Extensions requires a Git executable to be on your computer which does not appear to be the case. Please install Git or click Browse to find Git on your computer manually.
```


What's in the original message isn't sufficiently clear in your opinion?

Reply:
I just created the modifications based on the discussion over the issue #7726  itself. I don't have any strong opinion on either of these, I just tried to contribute to the application.

Reply:
Let's keep the short version, thank you.

Reply:
Why have you changed your mind, @RussKie? https://github.com/gitextensions/gitextensions/issues/7726#issuecomment-586212844
I think the first sentence will help users to understand the dependency better.
I agree, the second sentence is kind of redundant with the TaskDialog buttons.

Reply:
> Why have you changed your mind, @RussKie? [#7726 (comment)](https://github.com/gitextensions/gitextensions/issues/7726#issuecomment-586212844)

Because of how the dialog will look - putting myself in a shoes of an average user to me it looks scary with so much text:
![image](https://user-images.githubusercontent.com/20760348/95017066-3ff6ea80-0657-11eb-9cce-07ef29ffa43c.png)

With UI quite often it is hard to know what a result would be, and with that a number of iterations may be required... And according to Jeff Bezos [The Smart People Change Their Minds](https://techcrunch.com/2012/10/19/jeff-bezos-the-smart-people-change-their-minds/) 😄 

Also to me it feels the message has a lot of duplicating information, which can be removed without affecting the comprehension.

I'm happy to iterate over the wording, if you have ideas.

Reply:
```suggestion
        private readonly TranslationString _gitExecutableNotFoundText = new TranslationString("Git Extensions requires a Git executable but could not locate it.");
```
?

Reply:
👍